src/main/java/uk/gov/hmcts/reform/professionalapi/controller/OrganisationController.java

```diff
@@ -32,13 +35,16 @@
 
     private OrganisationCreationRequestValidator validator;
 
+    private PaymentAccountService paymentAccountservice;
+
+
     @ApiOperation("Creates an organisation")
     @ApiResponses({
-        @ApiResponse(
-            code = 200,
```

Comment:
Can yo please update the @ApiResponse code 200 to 201 for createOrganisation() method.

Reply:
I have included these changes in my latest commit. Please review. Thanks!